backend/app/api/api_v1/routers/summaries.py

```diff
@@ -22,6 +22,11 @@
     summary="Gets a summary of the documents associated with a country.",
```

Comment:
are we planning to remove the country endpoint when the FE has been updated?

Reply:
yes